3rdparty/halrvv/rvv071/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION ${MIN_VER_CMAKE} FATAL_ERROR)
+
+set(HAL_LIB_NAME "")
+
+set(HALRVV071_HAL_FOUND TRUE CACHE INTERNAL "")
+set(HALRVV071_HAL_VERSION "0.0.1" CACHE INTERNAL "")
+set(HALRVV071_HAL_LIBRARIES ${HAL_LIB_NAME} CACHE INTERNAL "")
+set(HALRVV071_HAL_HEADERS "halrvv071.hpp" CACHE INTERNAL "")
+set(HALRVV071_HAL_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}" CACHE INTERNAL "")
+
+configure_file("halrvv071.hpp" "${CMAKE_BINARY_DIR}/halrvv071.hpp" COPYONLY)
```

Comment:
This line is not needed - we use header files directly from the source directory.